pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTaskRestletResource.java

```diff
@@ -289,6 +291,7 @@ public StringResultResponse getTaskStateDeprecated(
 
   @POST
   @Path("/tasks/schedule")
+  @Authenticate(AccessType.UPDATE)
```

Comment:
create? another one of these create/update cases

Reply:
Yeah, as you and I mentioned earlier, some of these endpoints are tricky in terms of annotating with create or update. Here I thought this method just updates the state of the system by triggering tasks.